tests/services/test_load_db.py

```diff
@@ -0,0 +1,180 @@
+import pytest
+from pymodm import connect
+
+from app.db.mongo_model_definitions import Service, User
+from app.services.load_db import LoadDb
+
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
```

Comment:
Doing this should not happen here - tests should just be run in virtualenv (`pipenv run`) which will load `.env` variables. Generally in this project I don't think that `load_dotenv` should be used at all.